PowerUp/app/src/main/java/powerup/systers/com/AvatarRoomActivity.java

```diff
@@ -270,6 +188,55 @@ public void onClick(View v) {
         getmDbHandler().close();
     }
 
+    @Override
+    protected void onSaveInstanceState(Bundle savedInstanceState) {
+        super.onSaveInstanceState(savedInstanceState);
+        savedInstanceState.putInt("eyeNum", eye);
+        savedInstanceState.putInt("faceNum", face);
+        savedInstanceState.putInt("hairNum", hair);
+        savedInstanceState.putInt("clothNum", cloth);
+    }
+
+    @Override
+    protected void onRestoreInstanceState(Bundle savedInstanceState) {
+        super.onRestoreInstanceState(savedInstanceState);
+        eye = savedInstanceState.getInt("eyeNum");
+        face = savedInstanceState.getInt("faceNum");
+        cloth = savedInstanceState.getInt("clothNum");;
+        hair = savedInstanceState.getInt("hairNum");
+        resource_setter(eye,getResources().getString(R.string.eye));
+        resource_setter(face,getResources().getString(R.string.face));
+        resource_setter(hair,getResources().getString(R.string.hair));
+        resource_setter(cloth,getResources().getString(R.string.cloth));
+    }
+
+    public void resource_setter(int value,String type){
```

Comment:
Functions are named using camel case convention and not the _ division. Please fix.